src/libs/actions/Policy.js

```diff
@@ -539,6 +539,15 @@ function subscribeToPolicyEvents() {
     });
 }
 
+/**
+ * Checks if we have any errors stored within the POLICY_MEMBER_LIST.  Determines whether we should show a red brick road error or not
+ * @param {Array} policyMemberList
```

Comment:
I think this is wrong and this should be `Object`, since the members list is an object keyed by email

Reply:
Do you have an example of what this key looks like?

I thought the employee list would look like this since it also contains errors
```
[
 	{email: bla@gmail.com, errors: []},
	{email: bla1@gmail.com, errors: []}
]
```

Reply:
Per the policy refactor doc IIRC it was `{email: {role:'bla', bla: 1}, email2: {...},...}`